src/models/flight_control/FGLinearActuator.cpp

```diff
@@ -0,0 +1,252 @@
+            isResetProperty = true;
+        }
+
+        if (element->FindElement("versus")) versus = element->FindElementValueAsNumber("versus");
+        if (element->FindElement("lag")) lag = element->FindElementValueAsNumber("lag");
+        if (element->FindElement("module")) module = element->FindElementValueAsNumber("module");
+        if (element->FindElement("rate")) {
+            rate = element->FindElementValueAsNumber("rate");
+            if (rate <= 0 or rate > 1.0) rate = 0.5;
+            minRate = -(rate);
+            maxRate = rate;
+        }
+        if (element->FindElement("gain")) gain = element->FindElementValueAsNumber("gain");
+        
+        FGFCSComponent::bind();
+        
+        Debug(0);
+    }
+    
+    //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    
+    FGLinear_Actuator::~FGLinear_Actuator()
+    {
+        Debug(1);
+    }
+    
+    //%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+    
+    bool FGLinear_Actuator::Run(void )
+    {
+        Input = InputNodes[0]->getDoubleValue();
+        if (initialized) {
+            input_prec = Input;
+            initialized = 0;
+            std::cerr << "FGLinear_Actuator Cicle_zero " << InputNodes[0]->GetNameWithSign() << std::endl;
+            return true;
+        }
+        
+        Output = 0.0;
+        
+        set = 1.0;
+        reset = 0.0;
+        
+        if (isSetProperty) set = setProperty;
+        if (isResetProperty) reset = resetProperty;
+            
+        if (set > 0 ) {
+            
+            double input_delta = Input - input_prec;
+            
+            if ((versus < 0.0 and input_delta > 0.0) or (versus  > 0.0 and input_delta < 0.0)) {
+                input_delta = 0.0;
+            }
+            input_prec = Input;
+            
+            if (input_delta != 0.0) {
+                if (input_delta <= (module * maxRate) and input_delta >= (module * minRate)) {
+                    if (input_delta > 0.0) {
+                        direction = 1;
+                        //cout << "FGLinear_Actuator::Run" << InputNodes[0]->GetNameWithSign() << " direction + " << direction << endl;
+                    } else if (input_delta < 0.0) {
+                        direction = -1;
+                        //cout << "FGLinear_Actuator::Run" << InputNodes[0]->GetNameWithSign() << " direction - " << direction << endl;
+                    }
+                }
+                
+                if (direction != 0) {
+                    if (input_delta >= (module * maxRate) or input_delta <= (module * minRate)) {
+                        if (direction > 0) {
+                            countSpin++;
+                            cout << "FGLinear_Actuator::Run" << InputNodes[0]->GetNameWithSign() << " countSpin++ " << countSpin << " maxRate " << maxRate << " minRate " << minRate << endl;
```

Comment:
Don't issue a message at each time step since that will flood the terminal/console of the user.
```suggestion
```